src/monocle_apptrace/wrap_common.py

```diff
@@ -266,3 +272,35 @@ def update_tags(instance, span):
         span.set_attribute(TAGS, [model_name, vector_store_name])
     except:
         pass
+
+
+def update_vectorstore_attributes(to_wrap, instance, span):
+    """
+       Updates the telemetry span attributes for vector store retrieval tasks.
+    """
+    try:
+        # Check if the span is for a vector store retriever task from langchain
+
+        if to_wrap['package'] == 'langchain_core.retrievers':
```

Comment:
if we have to add more frameworks, how can we handle this in a configurable way ?

Reply:
@oi-raanne  updated the logic , can you review this ?

Reply:
done